rules/docker-telemetry.mk

```diff
@@ -27,5 +27,6 @@ endif
 $(DOCKER_TELEMETRY)_CONTAINER_NAME = telemetry
 $(DOCKER_TELEMETRY)_RUN_OPT += --net=host --privileged -t
 $(DOCKER_TELEMETRY)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
+$(DOCKER_TELEMETRY)_RUN_OPT += --mount type=bind,source="/var/platform/",target="/mnt/platform/"
```

Comment:
why do we need to mount platform in mgmt framework. it suppose to be platform independent, right?

Reply:
This is used in two places. 1. Get system information 2. If needed override the default validation(if it is deemed insufficient) and do platform specific validation

Reply:
what system information, we have put sys eeprom information in state db.

Reply:
Following link has the information that we get for the openConfig System module. This is the first module in the link below.

https://github.com/project-arlo/sonic-mgmt-framework/wiki/YANG-Object-Reference